### PR TITLE
Refactor StartState

### LINKED ISSUE #181: [PE-D] When I reopen the jar open after setting the active level to 0, all the input are considered invalid and i cant do anyting about it
-------------
Labels: `type.FunctionalityBug` `severity.High`
original: ruyian/ped#2

### LINKED ISSUE #182: [PE-D] The blank in name
After deleting the relevant files to fix the previous bug, the name input becomes blank

 while blank input should not be read into the system, evident in this

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: ruyian/ped#3

### LINKED ISSUE #185: [PE-D] Mistake in the help command when describing the format of view command
There are some mistakes in the instructions for the view command format (Exercise Bank and Food Bank)

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: dyahnafisah/ped#1

### PR DESCRIPTION
**Pull request by tttyyzzz**
Refactor StartState, modify help command, removed \t to avoid confusion, changed \n to LS update user guide accordingly.

Repeat user message on invalid input to prevent raging users.

Fixes #185 aligned help message and eliminate possible mistakes
Fixes #182 added .trim() in name class when comparing. Hope repo sense wont pick it up.
Fixes #181 not an error, is just raging tester. Reduced rage by printing prompt below invalid command.

### CODE REVIEW

File: src/main/java/seedu/duke/main/CreateActivityFactor.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package seedu.duke.main;
+
+import seedu.duke.data.profile.Profile;
+import seedu.duke.data.profile.attributes.ActivityFactor;
+import seedu.duke.data.profile.utilities.ProfileUtils;
+import seedu.duke.logic.parser.exceptions.ParamMissingException;
+import seedu.duke.ui.Ui;
+
```
Comment (weidak) [inline src/main/java/seedu/duke/main/CreateActivityFactor.java:8]: Remember to write JavaDocs for your classes

**Review by weidak — APPROVED** [review]
Good usage of OOP here! Looks very abstracted and clean 👍

File: src/main/java/seedu/duke/main/CreateActivityFactor.java (hunk 1 above)
Comment (tttyyzzz) [reply src/main/java/seedu/duke/main/CreateActivityFactor.java:8]: Thanks for reminding! will do it few hours later.

**Comment by tryyang2001** [comment]
Sorry there is a problem in my FoodListTest, I will merge my pr later and the problem will be solved.

**Pull request merged**